### ISSUE TITLE
Consider Windows specific section

### ISSUE THREAD

**Issue by bigos** [CodyReichert/awesome-cl#97]
It appears that some of your suggestions will not work on Windows. Although it is in theory possible to make it work on Windows though MSYS2 or similar, often it is not worth the trouble. For a brief moment, I had Qtools working on Windows perfectly, but after the recent big update or my changes, it seems impossible to make it work again. With the other contender namely, cl-cffi-gtk I had even less luck.

**Comment by rudolfochrist** [comment]
I'm not quite sure if I understand your question. Do you want a section that list libraries that only work on Windows or an additional section that list libraries that are problematic on Windows? 

Personally I don't use Windows so I can't verify what will be working on Windows and what not, but I'm sure if you start such section and open a PR it will be highly appreciated. 

If you're asking that this list should only contain libraries that are verified to work on Windows then I tell you that this is not what this list intended to be. 

Best,
Sebastian

**Comment by bigos** [comment]
Yes, this is a tricky question. I use Linux most of the time, but recently to my horror, I have discovered that many libraries that are important to me do not work on Windows. Perhaps adding a note that the list applies only to Mac or Linux and whenever external system libraries are involved Windows users should not be surprised if they experience troubles. The more I look at the problem, the more I am inclined to leave you alone and suggest to close the issue. Perhaps one day someone could have a Windows edition of your list. But creating it is a very tricky. Windows doesn't have a proper scripting environment. Solutions that try to fix it, like MSYS2 or similar are very fragile and providing any support to them doesn't make much sense. Also, you would need separate instructions for bare Windows, MSYS2, Cygwin or anything else that tries to fill the void.

**Comment by CodyReichert** [comment]
Ah interesting. That hasn't really crossed my mind since I primarily use Linux.

That said, I think there's a couple of things we could consider:

1. If the list of libraries that don't work on Windows isn't _huge_, we could just put a little note next to those items like "No Windows Support". We could do it incrementally as they pop up, then we don't need a separate list of possibly duplicate libraries.

2. If there are quite a few that don't work on Windows (like over 50%), I would be OK putting a note on the README that says "Many of these libraries only work on Linux/Mac and are no supported on Windows".

How many did you try that didn't work, @bigos?

Similarly, we also have a note that says "these libraries are available on QuickLisp unless otherwise stated", and I think that might have fallen out of date with some newer entries - so a process for knowing where to get a library and what platforms it supports would be nice.

But only if it's rampant, I think.

**Comment by bigos** [comment]
I only tried a few GUI libraries and none of them worked properly. I mean I had for a while almost working qtools and cl-cffi-gtk, but I have no idea why it has worked. After recent system upgrade it stopped working. Cermic installation failed due to problems with libuv library. So i theory they could be made to work. There is a [roblem with lack of properly supported solution. Msys2 with a bit of polish could be a good candidate. The problem is lots of advice about it on the internet is wrong. You are not supposed to modify the Windows paths in Control Panel to make it work with the rest of the system. Chocolatey might be another solution but i did not try it extensively. Windows is a pain to use if you come from Linux. At the moment to do my work I am facing prospects of using 3 different scripting systems!

**Comment by azzamsa** [comment]
Rather than adding new subsection, I recommend to use additional description or icon for "windows not supported". 
So it's only icon/description beside library name or in the end of library description.
